### PR TITLE
feat(relays): add export to clipboard and handle multiple relay urls

### PR DESCRIPTION
**Pull request by fernandolguevara**
image

### CODE REVIEW

File: packages/app/src/Pages/settings/Relays.tsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ const RelaySettingsPage = () => {
     }
   }
 
+  function exportRelays() {
+    navigator.clipboard.writeText(Object.keys(relays).join("\r\n"));
```
Comment (v0l) [inline packages/app/src/Pages/settings/Relays.tsx:35]: Maybe we can join them by one of the separators in the split below, why `\r\n`

File: packages/app/src/Pages/settings/Relays.tsx [hunk 2]
```diff
@@ -41,29 +45,72 @@ const RelaySettingsPage = () => {
...
+        </div>
       </>
     );
   }
 
   function addNewRelay() {
-    if ((newRelay?.length ?? 0) > 0) {
-      const parsed = new URL(newRelay ?? "");
+    if (newRelay?.length) {
+      let seps = "; ,".split("");
```
Comment (v0l) [inline packages/app/src/Pages/settings/Relays.tsx:68]: Doesn't seem necessary to support multiple separators, space alone seems fine

File: packages/app/src/Pages/settings/Relays.tsx (hunk 1 above)
Comment (fernandolguevara) [reply packages/app/src/Pages/settings/Relays.tsx:35]: I want the list to be splited by new lines, If I need to paste the it on x IM clients

File: packages/app/src/Pages/settings/Relays.tsx (hunk 2 above)
Comment (fernandolguevara) [reply packages/app/src/Pages/settings/Relays.tsx:68]: I prefer to use this list of characters for delimited lists

**Comment by v0l** [comment]
Rejected

**Comment by fernandolguevara** [comment]
joya aborto de nutria 🤙🏻

**Pull request closed without merging**